### PR TITLE
chore(cli): make the dashes under column headings consistent

### PR DESCRIPTION
**Pull request by huanjani**
Wherever command output is in tabular format, this brings consistency to our formatting-- dashes (same length as heading, not longest element) below the headings.

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

**Comment by huanjani** [comment]
Ex: 
```
% copilot svc status
Found only one deployed service acorn in environment envwvpc
Service Status

  ACTIVE 1 / 1 running tasks (0 pending)

Last Deployment

  Updated At         1 day ago
  Task Definition    arn:aws:ecs:us-west-2:2405947XXXXX:task-definition/sample-envwvpc-acorn:2

Task Status

  ID                Image Digest        Last Status         Started At          Stopped At          Health Status
  --                ------------        -----------         ----------          ----------          -------------
  57c42201          21797652            RUNNING             1 day ago           -                   UNKNOWN

Alarms

  Name              Condition           Last Updated        Health
  ----              ---------           ------------        ------
```
```
 % copilot svc ls    
Name                Type
----                ----
acorn               Load Balanced Web Service
```
```
% copilot job ls    
Name                Type
----                ----
job3                Scheduled Job
```
```
% copilot app show            
About

  Name              sample
  URI               

Environments

  Name              AccountID           Region
  ----              ---------           ------
  envwvpc           2405947XXXXX        us-west-2

Services

  Name              Type
  ----              ----
  acorn             Load Balanced Web Service

Pipelines

  Name
  ----
```

File: internal/pkg/cli/env_show_test.go [hunk 1]
```diff
@@ -285,7 +285,7 @@ func TestEnvShow_Execute(t *testing.T) {
...
+			wantedContent: "About\n\n  Name              testEnv\n  Production        false\n  Region            us-west-2\n  Account ID        123456789012\n\nServices\n\n  Name              Type\n  ----              ----\n  testSvc1          load-balanced\n  testSvc2          load-balanced\n  testSvc3          load-balanced\n\nTags\n\n  Key                  Value\n  ---                  -----\n  copilot-application  testApp\n  copilot-environment  testEnv\n  key1              value1\n  key2              value2\n\nResources\n\n  AWS::IAM::Role           testApp-testEnv-CFNExecutionRole\n  testApp-testEnv-Cluster  AWS::ECS::Cluster-jI63pYBWU6BZ\n",
```
Comment (efekarakus) [inline internal/pkg/cli/env_show_test.go]: 🤣 we should use the backticks to stop this madness haha

File: internal/pkg/describe/status.go [hunk 2]
```diff
@@ -195,12 +195,14 @@ func (s *ServiceStatusDesc) HumanString() string {
 	fmt.Fprint(writer, color.Bold.Sprint("\nTask Status\n\n"))
 	writer.Flush()
 	fmt.Fprintf(writer, "  %s\t%s\t%s\t%s\t%s\t%s\n", "ID", "Image Digest", "Last Status", "Started At", "Stopped At", "Health Status")
+	fmt.Fprintf(writer, "  %s\t%s\t%s\t%s\t%s\t%s\n", "--", "------------", "-----------", "----------", "----------", "-------------")
```
Comment (efekarakus) [inline internal/pkg/describe/status.go]: can we [strings.Repeat](https://golang.org/pkg/strings/#Repeat) to simplify our lives and possibly less prone to human error?
`strings.repeat("-", len("ID"))`

Tests can remain as is (manual dashes)

File: internal/pkg/cli/env_show_test.go (hunk 1 above)
Comment (huanjani) [reply internal/pkg/cli/env_show_test.go]: right! done.

File: internal/pkg/describe/status.go (hunk 2 above)
Comment (huanjani) [reply internal/pkg/describe/status.go]: yes! done.

**Review by efekarakus — APPROVED** [review]

File: internal/pkg/list/list_test.go [hunk 3]
```diff
@@ -36,7 +36,7 @@ func TestList_JobListWriter(t *testing.T) {
 			inputAppName:   mockAppName,
 			inputWriteJSON: false,
 
-			wantedContent: "Name                Type\n--------            -------------\nbadgoose            Scheduled Job\nfarmer              Scheduled Job\n",
+			wantedContent: "Name                Type\n----                ----\nbadgoose            Scheduled Job\nfarmer              Scheduled Job\n",
```
Comment (iamhopaul123) [inline internal/pkg/list/list_test.go]: Can we make the content easier to read as well? like https://github.com/aws/copilot-cli/pull/1705/files#diff-2cf2608db8d11464d4ffcc84647f98c75de2a474d1c6deadcedeae0806b4c0feR288-R316

Comment (huanjani) [reply internal/pkg/list/list_test.go]: yup, done.

**Comment by huanjani** [comment]
(I have a refactor planned after I merge #1704, so I'm sticking the do-not-merge label on this.)

**Review by iamhopaul123 — APPROVED** [review]

**Pull request merged**